interfaces/builtin/network_control.go

```diff
@@ -319,6 +320,10 @@ socket AF_NETLINK - NETLINK_GENERIC
 
 # for receiving kobject_uevent() net messages from the kernel
 socket AF_NETLINK - NETLINK_KOBJECT_UEVENT
+
+# For XDP:
+bpf
+socket AF_XDP
```

Comment:
I think there is no need to explicitly define the `socket AF_XDP` since it is already defined in https://github.com/snapcore/snapd/blob/master/interfaces/seccomp/template.go#L447

Reply:
This is fixed now, thank you